src/index.html

```diff
@@ -4,11 +4,22 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>To Do List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
     <br>
-    <div id="data">
-        <h1> to do list</h1>
+    <div class="container">
```

Comment:
- Optional 
I guess you left this button by mistake. Kindly remove this as this is not serving any functionality.
![image](https://user-images.githubusercontent.com/33268927/179160662-895a91c9-db37-4c44-944c-29ed5fd87a52.png)